python/cudf/cudf/core/subword_tokenizer.py

```diff
@@ -0,0 +1,279 @@
+        This can be created from the raw vocabulary
+        using the ``cudf.utils.hash_vocab_utils.hash_vocab`` function
+
+    do_lower : bool, Default is True
+        If set to true, original text will be lowercased before encoding.
+
+    Returns
+    -------
+    SubwordTokenizer
+    """
+
+    def __init__(self, hash_file: str, do_lower_case: bool = True):
+
+        self.do_lower_case = do_lower_case
+        self.vocab_file = cpp_hashed_vocabulary(hash_file)
+
+    def __call__(
+        self,
+        text,
+        max_length,
+        max_num_rows,
+        add_special_tokens=True,
+        padding="max_length",
+        truncation=False,
+        stride=0,
+        return_tensors="cp",
+        return_token_type_ids=False,
+    ):
+        """
+        Run CUDA BERT subword tokenizer on cuDF strings column.
+        Encodes words to token ids using vocabulary from a
+        pretrained tokenizer.
+
+        Parameters
+        ----------
+        text: (cudf String Series)
+            The sequence or batch of sequences to be encoded.
+            Each sequence can be a cudf string series
+
+        max_length: (int)
+            Controls the maximum length to use by one of
+            the truncation/padding parameters.
+
+        max_num_rows: (int)
+            Maximum number of rows for the output token-ids expected to
+            be generated by the tokenizer.
+            Used for allocating temporary working memory on the GPU device.
+            If the output generates a larger number of rows,
+            behavior is undefined.
+            This will vary based on stride, truncation, and max_length.
+            For example, for non-overlapping sequences output rows will be
+            the same as input rows.
+            A good default can be twice the max_length
+
+        add_special_tokens: (bool, optional, defaults to True)
+            Whether or not to encode the sequences with the special tokens
+            of the BERT classification model
+
+        padding: ('max_length')
+            Pad to a maximum length specified with the argument max_length
+
+        truncation: (bool, defaults to False)
+            True: Truncate to a maximum length specified with the
+              argument max_length.
+            False or 'do_not_truncate' (default)
+              No truncation (Output differs from HuggingFace)
+
+        stride (int, optional, defaults to 0)
+            The value of this argument defines the number of
+            overlapping tokens.
+            The information about the overlapping tokens is
+            present in the metadata outputed.
+
+        return_tensors (str) default (cp.ndarray objects)
+            'tf': Return TensorFlow tf.constant objects
+            'pt': Return PyTorch torch.Tensor objects
+            'cp': Return cupy cp.ndarray objects
+
+        return_token_type_ids (bool, optional): Only False currently supported
```

Comment:
I think these need a bit of formatting love. Needs to have a space before and after the colons for formatting reasons.

Also, is there a reason the types are in parenthesis?

Reply:
Fixed all documentation and formatting issues with commit https://github.com/rapidsai/cudf/pull/7942/commits/26cc440377ac3f6eaadb9597716e89b073490f25

This is how it is rendered locally: 
![image](https://user-images.githubusercontent.com/4837571/115787883-bb13ba00-a377-11eb-91d1-bb335de0e222.png)

![image](https://user-images.githubusercontent.com/4837571/115787916-c535b880-a377-11eb-9b6b-91f7981ce306.png)

![image](https://user-images.githubusercontent.com/4837571/115787941-cc5cc680-a377-11eb-93c4-e46cb97432c8.png)